etc/categories

```diff
@@ -128,6 +128,7 @@ Pycharm CE|Editors
 QEMU|Tools
 Quartz|Internet/Browsers
 Raspi2png|Tools
+ReaPack|Multimedia
```

Comment:
I think this change is unintended and unnecessary.

Reply:
Oh yeah, I accidentally based this branch off of the ReaPack pr one, so I had to remove ReaPack, but I guess when I did that, I missed the categories file..